### ISSUE TITLE
try simpler static obstacle scenario

### ISSUE THREAD

**Issue by sjwo** [sjwo/path_planner#15]
Try static obstacle scenario that is simpler than narrow_wide, with the latter of which the planner seems to take too long to return. We want to see some curvy lines around obstacles, but simple enough that the planner returns within the time bound every time.

Could do anything simpler, or could do Portsmouth harbor to Isles of Shoals.

If this is successful, then move on to dynamic obstacles.

If unsuccessful, then "Troubleshoot why controller has difficulty keeping the line" (backlog).

**Comment by sjwo** [comment]
Almost seems like the issue with BIT* on narrow_wide was that the turning radius of 8 m and the grid cell width of 5 m meant that the boat had to navigate through passage ways whose width was much less than the boat's turning radius. Going to try narrow_wide with resolution of 25 instead of 5, which is just over three times the default turning radius of 8 m.

**Comment by sjwo** [comment]
Just tried narrow_wide-25.test with RBPC, and it still crashed on the west wall just after turning into the passage. And I think I DO see the same line-keeping issues as I do with BIT*.

Tried it again and it crashed right outside the entrance, on the west corner of the opening.

**Comment by sjwo** [comment]
- [ ] Does the rqt > mission_manager > planner setting actually make a difference? Right now, it says PathFollower -- but I assumed that was overridden and/or ignore when running the test_scenario_runner...

Just used GUI to change this setting from PathFollower to PathPlanner, and it didn't crash at the channel opening, but it still looked very wiggly on its way there. Then it got stuck in local minimum of the U-shaped wave break.

- [ ] Perhaps I could add some output to PathFollower, so that I know when it's running?

**Comment by sjwo** [comment]
- [ ] Does test_scenario_runner circumvent the mission_manager?

**Comment by sjwo** [comment]
Running narrow_wide-25 with BIT*. Planner failed to return within time bound several times in a row and boat just drifted. Time horizon (irrelevant to BIT*) keeps getting halved. I guess narrow_wide-25 is still to complex for my as-yet slow BIT* planner. Things stopped when the boat drifted into the static obstacle and the collision was detected.

**Comment by sjwo** [comment]
So, looks like both RBPC and BIT* crash on original narrow_wide test with no_noise scenario configuration. Videos are here:

- RBPC: https://www.cs.unh.edu/~sjw1000/narrow_wide-rbpc-crash.mp4
- BIT*: https://www.cs.unh.edu/~sjw1000/narrow_wide-bitstar-crash.mp4

And it DOES look like RBPC has difficulty with line keeping the way BIT* does.

It also looks like narrow_wide is not on Alex's [YT video playlist of planner testing](https://www.youtube.com/playlist?list=PLNcViETgkSdceqn0dB9kG2_fTC_lEgDra), suggesting that the system may not have been known to work on this test case. This suggests that it would be better to test BIT* against a scenario where RBPC is known to have worked.

Alternatively, maybe I have messed something up with the Executive, making trouble for RBPC...
- [ ] checkout path_planner and mpc and test_scenario_runner from before I started making modifications, and see if RBPC still has trouble on `narrow_wide.test`.
